src/renderer/components/widgets/channels/ChannelMessage.js

```diff
@@ -276,70 +276,91 @@ export const ChannelMessage = ({
     }
   }, [imageUrl])
   const [actionsOpen, setActionsOpen] = useState(false)
+  const arrayOfGroupedMessages = [[]]
+  for (const el of parsedMessage) {
```

Comment:
we shouldn't do this logic in render;

imo move this to the effect that is calling setParsedMessage